lib/Map/VariableConcept.js

```diff
@@ -62,7 +62,14 @@ var VariableConcept = function(name, options) {
     knockout.getObservable(this, 'isActive').subscribe(function(nowActive) {
         // If this concept has been activated, and its parent has a color callback, use it to set this concept's color now.
         if (nowActive && defined(this.parent) && !defined(this.color) && defined(this.parent.getColorCallback)) {
-            this.color = this.parent.getColorCallback();
+            var that = this;
+            var columnId;
+            for (var i=0; i<this.parent.columns.length; i++) {
+                if (that.id === this.parent.columns[i].id) {
+                    columnId = i;
+                }
+            }
```

Comment:
I'd prefer not to have this object loop through its siblings in this way.  Not sure what the right way is though - let me think about it for a bit.